spec/classes/server/conf_spec.rb

```diff
@@ -1,48 +1,21 @@
 require 'spec_helper'
-
-file_content_7 = "/usr/bin/systemctl restart rsyslog > /dev/null 2>&1 || true"
-file_content_6 = "/sbin/service rsyslog restart > /dev/null 2>&1 || true"
-
 describe 'simp_openldap::server::conf' do
   context 'supported operating systems' do
     on_supported_os.each do |os, facts|
       context "on #{os}" do
         let(:facts) do
-          facts[:server_facts] = {
-            :servername => facts[:fqdn],
-            :serverip   => facts[:ipaddress]
+          facts[:slapd_version] = '2.4.40'
+          facts[:server_facts]  = {
+            :servername    => facts[:fqdn],
+            :serverip      => facts[:ipaddress],
           }
           facts
         end
 
-        let(:pre_condition) {
-          %(
-            class { "::simp_openldap":
-              base_dn   => "DC=host,DC=net",
-              is_server => true
-            }
-          )
-        }
-
-        context 'with default parameters' do
-          it { is_expected.to create_class('simp_openldap::server') }
-          it { is_expected.to create_class('simp_openldap::server::conf::default_ldif') }
+file_content_7 = "/usr/bin/systemctl restart rsyslog > /dev/null 2>&1 || true"
```

Comment:
Why are these out-dented?  Should then be global variables defined prior to the describe on line 2?  Should they be let() instances?